examples/siro_sandbox/test_sandbox_app.py

```diff
@@ -0,0 +1,79 @@
+import os
+from unittest.mock import patch
+
+import magnum as mn
+import pytest
+from sandbox_app import (
+    DEFAULT_CFG_PATH,
+    SandboxDriver,
+    get_sandbox_app_args_parser,
+    override_config,
+)
+from utils.gui.gui_input import GuiInput
+from utils.gui.replay_gui_app_renderer import ReplayGuiAppRenderer
+
+from habitat_baselines.config.default import get_config as get_baselines_config
+
+
+@pytest.mark.skipif(
```

Comment:
I'm curious if we can run this as part of CI. I started a slack thread [here](https://cvmlp.slack.com/archives/C0460NTKM4G/p1698436827946659).

Reply:
As Andrew replied the Slack tread `this assets are actually already being downloaded in the multi-agent trainer CI`.

Reply:
Cool. Can you link to a successful CI run of this file here as a reply? It's not clear to me at a glance if this file is getting included in the tests as part of CI `install_and_test_ubuntu`.